Translations/fr.po

```diff
@@ -2173,19 +2172,19 @@ msgstr "Potion de Mana Totale"
 
 #: Source/itemdat.cpp:84 Source/itemdat.cpp:441
 msgid "Griswold's Edge"
-msgstr "Epée de Griswold"
+msgstr "Épée de Griswold"
 
 #: Source/itemdat.cpp:85 Source/itemdat.cpp:442
 msgid "Bovine Plate"
-msgstr "Plate Bovine"
+msgstr "Plaque Bovine"
```

Comment:
I prefere the previous translation as it better refer to a armor IMO (https://fr.wikipedia.org/wiki/Plate_(armure))